packages/react-native/package.json

```diff
@@ -81,6 +81,7 @@
     "gradle.properties",
     "gradle/libs.versions.toml",
     "index.js",
+    "index.flow.js",
```

Comment:
I think it should be `index.js.flow`

Reply:
You're absolutely right, thanks for spotting this. Do you want to open a PR and tag me?